guides/v3.4.0/models/index.md

```diff
@@ -1,5 +1,19 @@
-Models are objects that represent the underlying data that your
-application presents to the user. Different apps will have very
+Ember developers have great options for how they handle data from back end APIs. 
+Ember itself works with any type of back end: REST, JSON API, GraphQL, or anything else.
+
+Many developers choose to use Ember Data, a powerful set of tools for formatting requests, serializing responses, and efficiently managing a local cache of data. 
```

Comment:
> for formatting requests, serializing responses, and efficiently managing a local cache of data. 

Typically outbound (from `ember-data`'s store) is "serialization" and inbound (to `ember-data`'s store) is "normalization". Alternatives to "normalization" include `deserialization` or language around `munging` `aligning` or `formatting` responses into `JSON API`.

Reply:
Fixed. I'll call it normalization here. I think that term has enough traction outside of Ember Data.